### PR TITLE
Vet clinic database: query multiple tables

### PR DESCRIPTION
**Pull request by cynthiainga**
### In this Milestone, I do the following tasks:

- [ ] Use primary key & foreign key mechanisms for joining tables.

- [ ] Query multiple tables.

- [ ] Prepare complex queries that answer analytical questions.

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -26,4 +26,36 @@ CREATE TABLE species(
...
+REFERENCES
+  owners(id);
+
+ALTER TABLE
+  animals
+ADD CONSTRAINT
+  fk_species
+FOREIGN KEY(species_id)
+REFERENCES
+  owners(id);
```
Comment (Kingstalux) [inline schema.sql]: - [ ] Good job so far 👍 This foreign key here should reference the `species(id)` and not the `owners(id)`. Kindly rectify this issue 😄

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @cynthiainga, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f6e59597451644c7177776c54653363634b682f67697068792e676966



## Highlights 👏 
✅ Good commit messages 👍 
✅ Good job adding and filling in the necessary tables 👏 
✅ Good job answering all the queries 🥂 



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (cynthiainga) [reply schema.sql]: Thanks @Kingstalux I did the confusion :smile: , thanks for the review :pray:

**Review by Kingstalux — APPROVED** [review]
Hi @cynthiainga,
# Status: Approved ✔️ ✔️ ✔️ 
 
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job implementing the required changes 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by cynthiainga** [comment]
Thank you :blush:

**Pull request merged**